examples/vello_editor/src/main.rs

```diff
@@ -201,6 +208,7 @@ fn main() -> Result<()> {
         state: RenderState::Suspended(None),
         scene: Scene::new(),
         editor: text::Editor::default(),
+        last_drawn_generation: Default::default(),
```

Comment:
SUGGESTION!

Maybe `last_drawn_generation: Generation::ZERO` and then where Daniel suggested `Generation(1)`, you'd use `Generation::INITIAL` and those are 2 pub consts on `Generation` that can be documented so that the usage and intent is clear.

But I'd get someone else to say that makes sense before you take it too seriously.

Reply:
I kinda like the idea, but also I think that `Default` is fine here, and `Generation(1)` is fine in editor.rs because (for now) it lives in the editor module, has no other users, and is generally not intended to be constructed anywhere else.